packages/components/src/components/hds/flyout/body.ts

```diff
@@ -7,15 +7,18 @@ import Component from '@glimmer/component';
 import { deprecate } from '@ember/debug';
 
 export interface HdsFlyoutBodySignature {
+  Args: {
+    contextualClass?: string;
+  };
```

Comment:
[praise] 🙌

Reply:
question: hm.. the `hds/flyout/body` doesn't have or make use of any `contextualClass`; `hds/dialog-primitive/body` has a `contextualClass`. unless I'm missing something all these signature additions are not needed in `flyout` and `modal`.

Reply:
We are passing `contextualClass` in as an argument in the index.hbs for those components.

```
  <:body>
    {{yield (hash Body=(component "hds/dialog-primitive/body" contextualClass="hds-modal__body"))}}
  </:body>
  ```
  
If we're not using those classes we can either remove them, or actually implement them. If they're not doing anything, I would lean toward removing them.